### ISSUE TITLE
--stats not showing query time

### ISSUE THREAD

**Issue by godlikeanyone** [natesales/q#27]
Hi, the document states that " --stats              Show time statistics". However, --stats did not show any additional information.

```
PS F:\Software\q_0.8.0_windows_amd64> ./q -q xp.apple.com -s quic://b.iqiq.io:784 --stats
xp.apple.com. 58m42s CNAME xp.itunes-apple.com.akadns.net.
xp.itunes-apple.com.akadns.net. 3m42s CNAME ioshost.qtlcdn.com.
ioshost.qtlcdn.com. 1m0s A 111.31.17.153
ioshost.qtlcdn.com. 1m0s A 122.70.142.168
xp.apple.com. 58m59s CNAME xp.itunes-apple.com.akadns.net.
xp.itunes-apple.com.akadns.net. 3m59s CNAME ioshost.qtlcdn.com.
xp.apple.com. 5m28s CNAME xp.itunes-apple.com.akadns.net.
xp.itunes-apple.com.akadns.net. 2m51s CNAME xp.apple.com.edgekey.net.
xp.apple.com.edgekey.net. 5m28s CNAME e17437.dsct.akamaiedge.net.
xp.apple.com. 5m28s CNAME xp.itunes-apple.com.akadns.net.
xp.itunes-apple.com.akadns.net. 2m51s CNAME xp.apple.com.edgekey.net.
xp.apple.com.edgekey.net. 5m28s CNAME e17437.dsct.akamaiedge.net.
xp.apple.com. 5m28s CNAME xp.itunes-apple.com.akadns.net.
xp.itunes-apple.com.akadns.net. 2m51s CNAME xp.apple.com.edgekey.net.
xp.apple.com.edgekey.net. 5m28s CNAME e17437.dsct.akamaiedge.net.
xp.apple.com. 37m3s CNAME xp.itunes-apple.com.akadns.net.
```

**Comment by dsully** [comment]
+1 - I was about to file the same issue.

**Comment by dsully** [comment]
Looks like stats are only displayed when the output format is `raw`.

**Comment by natesales** [comment]
Apologies for the delay here - I've just addressed this in 18d3af5 and released in v0.8.1.
